ipython-notebook-example/setup_env.sh

```diff
@@ -0,0 +1,11 @@
+#!/bin/bash
+OS_NAME="$(uname -s)"
+if [ '$OS_NAME' == 'Darwin' ]; then
+  brew install gfortran
```

Comment:
I bet it is scipy...

On Thu, Jan 14, 2016 at 12:33 PM, Alexandra L Johnson <
notifications@github.com> wrote:

> In ipython-notebook-example/setup_env.sh
> <https://github.com/sigopt/sigopt-examples/pull/7#discussion_r49781057>:
>
> > @@ -0,0 +1,11 @@
> > +#!/bin/bash
> > +OS_NAME="$(uname -s)"
> > +if [ '$OS_NAME' == 'Darwin' ]; then
> > +  brew install gfortran
>
> @iandewancker <https://github.com/iandewancker> can you comment?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/sigopt/sigopt-examples/pull/7/files#r49781057>.
>



-- 
Scott Clark
Co-founder and CEO
SigOpt, Inc. - www.sigopt.com - @SigOpt
scott@sigopt.com - @DrScottClark